ui.js

```diff
@@ -17,4 +17,5 @@
  *
  */
 
-module.exports = require("./dist/commands");
+module.exports = require("./dist/ui");
+
```

Comment:
Seems like there is an extra line break here. Please validate.

Reply:
Fixed with https://github.com/wso2/identity-cypress-test-base/pull/13/commits/99af09763311ab7271e55b85b9dbd83d84f2c112